### PR TITLE
Be able to use Redis sentinels

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: c2cwsgiutils/redis_utils.py [hunk 1]
```diff
@@ -1,13 +1,65 @@
...
+            print(_sentinel.sentinels[0].sentinel_masters())
+            raise Exception(_sentinel.sentinels[0].sentinel_masters())
+    if url:
+        if not url.startswith("redis://"):
+            url = "redis://" + url
+
+        LOG.info("Redis setup using: %s", url)
+        _master = redis.Redis.from_url(url, socket_timeout=socket_timeout, decode_responses=True)
+        _slave = _master
+    LOG.info("No Redis configuration found, use %s or %s to configure it", REDIS_URL_KEY, REDIS_SENTINELS_KEY)
```
Comment (arnaud-morvan) [inline c2cwsgiutils/redis_utils.py:61]: Missing else or return statement.
Seems we pass on this LOG even when we pass in ```if url```

File: scripts/publish [hunk 2]
```diff
@@ -33,3 +33,8 @@ then
   docker push camptocamp/c2cwsgiutils:${tag}
   rm -rf ~/.docker*  # docker logout
 fi
+
+if [[ $1 == refs/heads/master ]]
+then
+  docker push camptocamp/c2cwsgiutils-redis-sentinel:5
```
Comment (arnaud-morvan) [inline scripts/publish:39]: Need to login on docker hub first or I miss something.

Comment (arnaud-morvan) [reply scripts/publish:39]: Note that docker login could be factorized

File: c2cwsgiutils/redis_utils.py (hunk 1 above)
Comment (sbrunner) [reply c2cwsgiutils/redis_utils.py:61]: Thanks :-)

File: scripts/publish (hunk 2 above)
Comment (sbrunner) [reply scripts/publish]: +1 thanks :-)

**Pull request merged**